Air Conditioning/Daikin RTD-NET FBA-A(9).xml

```diff
@@ -27,9 +27,10 @@
     </RevisionHistory>
     <Name>ModbusModule-1</Name>
     <CcuModel>ModbusRtuCCU</CcuModel>
-    <Producer></Producer>
-    <Model></Model>
+    <Producer>Daikin</Producer>
+    <Model> RTD-NET FBA-A(9)</Model>
     <Description></Description>
+	<Category>3</Category>
     <Module>
       <Name>ModbusModule-1</Name>
```

Comment:
Premenuj to na nieco rozumnejsie ako ModbusModule-1

Reply:
Podľa čoho to mám pomenovať ?